rotkehlchen/exchanges/binance.py

```diff
@@ -129,8 +125,8 @@ def trade_from_binance(
     binance_pair = binance_symbols_to_pair[binance_trade['symbol']]
     timestamp = deserialize_timestamp_from_binance(binance_trade['time'])
 
-    base_asset = asset_from_binance(binance_pair.binance_base_asset)
-    quote_asset = asset_from_binance(binance_pair.binance_quote_asset)
+    base_asset = binance_pair.base_asset
```

Comment:
Moving the deserialization out of here is why you got hit by the cyclic import problems